opencv_warco/package.xml

```diff
@@ -10,7 +10,7 @@
   <!-- Commonly used license strings: -->
   <license>MIT</license>
 
-  <url type="website">https://github.com/lucasb-eyer/opencv-warco</url>
+  <url type="website">https://github.com/lucasb-eyer/opencv_warco</url>
```

Comment:
Damn... grep and sed have a mind on their own ;)
Will change in the next iteration, if I don't forget.